lib/Normalize_extended_ast.mli

```diff
@@ -9,6 +9,8 @@
 (*                                                                        *)
 (**************************************************************************)
 
+val ast : 'a Extended_ast.t -> ignore_doc_comments:bool -> Conf.t -> 'a -> 'a
```

Comment:
This seems unnecessary. Deleting it didn't affect building.

Reply:
Whoops, that was there for debugging – I’ll delete it.